### ISSUE TITLE
Tools appear but no image is visible

### ISSUE THREAD

**Issue by yormatt** [lunapaint/vscode-luna-paint#148]
Running Ubuntu 20. When I open PNG or JPG files the tool palettes appear but no image is ever visible.
![image](https://user-images.githubusercontent.com/32655284/180112809-1ac3ad64-25cb-416b-933e-e136ea008486.png)

**Comment by Tyriar** [comment]
It looks like you're on an old version looking at the tools window, could you try updating? You may need a newer version of vscode to get the latest version.

**Comment by yormatt** [comment]
I updated to buntu 1.69.2. Now I don't see any tools at all.

I tried reinstalling Luna and restarting Ubuntu but those made no
difference.


 Matt Morrise
E: ***@***.***
C: (503) 544-5105

**Comment by Tyriar** [comment]
Odd, could you open devtools (Help > Toggle Developer Tools) and check the console when you open up Luna Paint?

**Comment by player-001** [comment]
i actually have the same error with the latest version in which when i load an image nothing shows up, also i noticed that during the first launch of luna after installation the image loads and luna works amazingly but fails to load any other images after that and shows blank screen

HERE IS THE CONSOLE OUTPUT---
![22](https://user-images.githubusercontent.com/88928100/209705176-50f380a1-ecb9-4355-8b28-96d2d7d0965d.PNG)

**Comment by Tyriar** [comment]
@player-001 thanks, the error logs in particular are very handy. I can reproduce now if I add this setting which uses transparency for the editor bg:

```json
  "workbench.colorCustomizations": {
    "editor.background": "#11111180"
  }
```

**Comment by player-001** [comment]
@Tyriar yes you are right changing the background to a solid color without transparency fixed the issue
i guess i can use this as a temporary fix, thank you so much for you help

**Comment by Tyriar** [comment]
Great 🙂, not sure when the release will be but it's fixed for 0.17
